packages/trader/src/Services/Helpers/process-data.js

```diff
@@ -1,10 +0,0 @@
-/**
```

Comment:
Also for this file, why we removed this file?

Reply:
helpers added to shared package services

Reply:
as we talked because they are orphan files i removed them again.